packages/docmanager-extension/src/index.ts

```diff
@@ -410,14 +410,61 @@ export const downloadPlugin: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * A plugin providing open-browser-tab commands.
+ *
+ * This is its own plugin in case you would like to disable this feature.
+ * e.g. jupyter labextension disable @jupyterlab/docmanager-extension:open-browser-tab
+ *
+ * Note: If disabling this, you may also want to disable:
+ * @jupyterlab/filebrowser-extension:open-browser-tab
+ */
+export const openBrowserTabPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/docmanager-extension:open-browser-tab',
+  autoStart: true,
+  requires: [ITranslator, IDocumentManager],
+  optional: [],
```

Comment:
Looks like this could actually be removed if empty?

```suggestion
```

Reply:
Thanks, I updated the code with the change, then re-ran it, and it appears to be working fine. i.e. Open in new browser tab works as expected with the plugin active.